WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/AuthorsUseCase.kt

```diff
@@ -148,6 +150,17 @@ constructor(
         return items
     }
 
+    private fun getBarWidth(
```

Comment:
Do you think we can extract this function to some Utils class, as we are using it all around the place?

Reply:
I was thinking about it :-), I think it makes sense